openfecli/commands/gather.py

```diff
@@ -355,6 +355,30 @@ def _write_raw(legs:dict, writer:Callable, allow_partial=True) -> None:
                         m, u = format_estimate_uncertainty(m.m, u.m)
                     writer.writerow([simtype, *ligpair, m, u])
 
+def _check_legs_have_sufficient_repeats(legs):
+    """Throw an error if all legs do not have 2 or more simulation repeat results"""
+
+    bad_legs = []
+    for lig_pair, leg in legs.items():
+        for run_type, sim_results in leg.items():
+            n_repeats = len(sim_results)
+            if n_repeats < 2:
+                bad_legs += [(*lig_pair, run_type, n_repeats)]
+
+    if bad_legs:
+        # TODO: improve this formatting
+        msg = (
+            "ERROR: Every edge must have at least two simulation repeats.\n"
+            "The following legs have an insufficient number of repeats:\n\n"
+            "ligand_i,ligand_j\trun_type: n_repeats_found\n"
```

Comment:
something about "for a --dg calculation" I think we need to communicate that other reports will work or something

Reply:
good point! added.